sdk/keyvault/azure-security-keyvault-secrets/src/secret_client.cpp

```diff
@@ -9,12 +9,11 @@
 #include "azure/keyvault/secrets/secret_client.hpp"
 #include "azure/keyvault/secrets/keyvault_operations.hpp"
 #include "private/keyvault_protocol.hpp"
+#include "private/keyvault_urlScope.hpp"
```

Comment:
Please use snake_casing and not camelCasing for file names. If there are other occurrences in our codebase (I saw one earlier), please fix them too.
```suggestion
#include "private/keyvault_url_cope.hpp"
```